### DISCUSSION TITLE
Manutenzione lato admin

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#395] [Idee e proposte]
Buongiorno, quando si effettua la manutenzione all'interno del registro elettronico, invece di impedire a tutti gli utenti l'accesso allo stesso fino alla riapertura stabilita ,sarebbe possibile indicare la categoria di utenti che vi possono accedere o meno?Lo chiedo perché dovevo effettuare dei lavori di manutenzione e non posso accedervi nemmeno come admin fino a domattina

**Comment by trinko** [comment]
La segno come richiesta.
